### DISCUSSION TITLE
What is the reason for removing NVCleanstall from the repository?

### DISCUSSION THREAD

**Discussion by uffemcev** [microsoft/winget-pkgs#105953] [Q&A]
Hello, I'm not sure if this is the right place for that question, sorry. I would like to know why NVCleanstall was removed from the repository?

**Comment by mdanish-kh** [comment]
The reason was that InstallerUrl returned a 404 when run accessed in a browser and was removed in the current wave of removing packages with 404 Urls. However, I believe this was a false positive because of https://github.com/microsoft/winget-pkgs/issues/70469#issuecomment-1218684399. I'll open a PR adding the package back.

cc @Trenly - you may want to exclude TechPowerUp packages from your 404 removal script in the future.

**Reply by Trenly** [reply]
Thanks for the note!

**Comment by mdanish-kh** [comment]
- #105963
- #105964
- #105955

Will be resolved once these PRs get merged
